packages-ts/gauntlet-terra-cw-plus/src/commands/multisig.ts

```diff
@@ -6,6 +6,12 @@ import { isDeepEqual } from '../lib/utils'
 import { fetchProposalState, makeInspectionMessage } from './inspect'
 import { Vote, Cw3WasmMsg, Action, State, Cw3BankMsg } from '../lib/types'
 
+export const multisig: AccAddress = ((): AccAddress => {
```

Comment:
This address, the CW3_FLEX_MULTISIG address, and all other addresses will come from a specific source.
The source can be anything but will be RDD `json` file in most cases, so we'll have a loader for that type of source.

I'm commenting on this to share info about how we will be removing addresses from `.env` files in the short term.

So I'd suggest that [commands/multisig.ts](https://github.com/smartcontractkit/chainlink-terra/pull/204/files#diff-bd07cd3050690d85ba93f4ad25da078a48adfaac28dc26534d4d1766ffbda81e) has no purpose of knowing and exporting the mutisig address. The multisig address will be injected to the input of the command.